README.md

```diff
@@ -17,41 +17,50 @@ Create a `.env` file:
 cp .env.sample .env
 ```
 
-### direnv setup and configuration
+### Installing and Configuring `direnv`
+
+We use [`direnv`](https://direnv.net/) to automatically set environment variables on a per-project basis. The `.envrc` file found in the project's directory tells `direnv` to read in the environment variables specified in the `.env` file.
+
+Follow these steps to set up `direnv` on your machine:
 
 1. [Install direnv (`brew install direnv`)](https://direnv.net/docs/installation.html)
 2. [Add a hook for your shell](https://direnv.net/docs/hook.html)
 3. Restart your terminal
 4. Run `direnv allow` in the project directory
 
-### Database setup
+When navigating into the `phinvads-go` project directory, you should see something like this in your terminal, which lets you know that the environment variables are loaded:
 
-#### Running Postgres in Docker
+```bash
+direnv: export +DB_HOST +DB_NAME +DB_PASSWORD +DB_PORT +DB_USER +HOST +PORT
+```
 
-Download and install [Docker](https://www.docker.com/products/docker-desktop/) if you don't already have it installed.
+### Local Development
 
-1. Place `phinvads.dump` into the top level of your project directory (`/phinvads-go`)
-2. Navigate to the project directory and start your PostgreSQL database with `make start`
-3. Run `make refresh` to create the `phinvads` database and load in the data
+In this section, we'll:
 
-When you want to shut down your database environment, run `make stop`.
+1. Start PostgreSQL in a container or locally
+2. Load data into PostgreSQL
+3. Start the app
 
-#### Running Postgres locally
+#### Running PostgreSQL in a Container
 
-1. Install and run [PostgreSQL](https://www.postgresql.org/download/)
-1. Create an empty database:
+You may choose to manage your database using Docker. `phinvads-go` provides a [Docker Compose](https://docs.docker.com/compose/) file to allow for a streamlined database setup. Please download and install [Rancher Desktop](https://rancherdesktop.io/) if you don't already have it installed.
 
-    ```bash
-    psql -c 'CREATE DATABASE phinvads'
-    ```
+Note: If you'd prefer to install PostgreSQL locally, please skip the rest of this section and refer to the [Local Database Setup](#local-database-setup) section instead.
 
-1. Load the database dump file:
+To start your database, run this command:
 
-    ```bash
-    pg_restore -d phinvads --no-owner --role=$(whoami) phinvads.dump
-    ```
+`make startdb`
 
-### Application setup
+Next, please follow these steps in order to load in the data:
+
+1. Place `phinvads.dump` into the top level of your `/phinvads-go` project directory (Please let an engineering team member know if you need a copy of `phinvads.dump`)
+2. Navigate to the project directory and start your PostgreSQL database with `make startdb` (if the container isn't already running)
+3. Run `make refresh` to create the `phinvads` database and load in the data
```

Comment:
should we rename `refresh` to `refreshdb` now that all the other make commands have a post-fix noun?

Reply:
Yes, good idea!